src/Aspire.Cli/Commands/UpdateCommand.cs

```diff
@@ -94,4 +116,90 @@ private static bool UsesCentralPackageManagement(FileInfo projectFile)
 
         return false;
     }
+
+    private async Task<int> DetectAndWarnAboutAspireWorkloadAsync(CancellationToken cancellationToken)
```

Comment:
If this fails, we want to skip it. Don't fail update because we couldn't detect the workload.